client/ayon_nuke/startup/custom_write_node.py

```diff
@@ -90,15 +90,17 @@ def process(self):
             else:
                 knobs = node_knobs_presets
 
-        ext_knob_list = [knob for knob in knobs if knob["name"] == "file_type"]
-        if not ext_knob_list:
+        knob_names = {knob["name"]: knob for knob in knobs}
+
+        if "ext" in knob_names:
+            ext = knob_names["ext"]["value"]
+        elif "file_type" in knob_names:
+            ext = knob_names["ext"]["value"]
+        else:
             nuke.message(
-                "ERROR: No file type found in the subset's knobs."
+                "ERROR: No 'file_type' nor 'ext' found in the subset's knobs."
```

Comment:
Should this `subset` be refactored to `product` now too? While we're at it.

```suggestion
                "ERROR: No 'file_type' nor 'ext' found in the product's knobs."
```

Reply:
sounds like a good idea ideed !